Package.swift

```diff
@@ -1,10 +1,10 @@
-// swift-tools-version:5.5
+// swift-tools-version:5.7
```

Comment:
Why such an old Xcode version?

Reply:
Didn't know that was an xcode version to be honest. I updated the minimum version to v13 and it said this has to be at least 5.7, so I changed it to fix

I'll update to 6.0.2

Reply:
Make it "5.11". 6 will trigger a lot of concurrency things we don't want to deal with yet.